app/src/main/assets/pdf/vendor/pdf.js/pdf_viewer.css

```diff
@@ -0,0 +1,338 @@
+/* Copyright 2014 Mozilla Foundation
```

Comment:
Is there a minified version of this file? We could put the minified version on release build and this version on debug build so that we can read the code during development but when we ship the size is as small as it can be.

Also we can add version suffix to this file so we know what version of PDF.js we're using.